RobotArmAPP/Views/Controller.xaml.cs

```diff
@@ -0,0 +1,392 @@
+        }
+        #endregion
+
+        #region SPEED/REPETITIONS/DELAY
+        private void FrameSpeedBox_KeyDown(object sender, KeyRoutedEventArgs e)
+        {
+            controls.CheckBoxOnlyNumber(e);
+        }
+
+        private void FrameSpeedBox_KeyUp(object sender, KeyRoutedEventArgs e)
+        {
+            controls.BoxFocusOut(e, ControllerPage);
+        }
+
+        private async void FrameSpeedBox_LostFocus(object sender, RoutedEventArgs e)
+        {
+            await controls.BoxesMaxNumberLimiter(1, RepeatTimesBox, FrameSpeedBox, DelayBox);
+        }
+
+
+        private void DelayBox_KeyDown(object sender, KeyRoutedEventArgs e)
+        {
+            controls.CheckBoxOnlyNumber(e);
+        }
+
+        private void DelayBox_KeyUp(object sender, KeyRoutedEventArgs e)
+        {
+            controls.BoxFocusOut(e, ControllerPage);
+        }
+
+        private async void DelayBox_LostFocus(object sender, RoutedEventArgs e)
+        {
+            await controls.BoxesMaxNumberLimiter(2, RepeatTimesBox, FrameSpeedBox, DelayBox);
+        }
+
+
+        private async void MinDelay_Click(object sender, RoutedEventArgs e)//CALCULADO APENAS PARA O MG995
+        {
+            controls.SwitchMinimizeDelay(1, FrameSpeedBox, FramesListView, framesList);
+            await jsonSaverAndLoader.JsonAutoSaver(framesList, RepeatTimesBox.Text);
+        }
+
+        private async void MinDelayAll_Click(object sender, RoutedEventArgs e)
+        {
+            changingControls = true;
+            controls.LockControls(true, true, Blocker1, Blocker2, Blocker3);
+            Canvas.SetZIndex(StopPlayback, 1);
+            Canvas.SetZIndex(Blocker2, 2);
+
+            controls.SwitchMinimizeDelay(0, FrameSpeedBox, FramesListView, framesList);
+
+            changingControls = false;
+            controls.LockControls(false, false, Blocker1, Blocker2, Blocker3);
+
+            await jsonSaverAndLoader.JsonAutoSaver(framesList, RepeatTimesBox.Text);
+        }
+
+
+        private void RepeatTimesBox_KeyDown(object sender, KeyRoutedEventArgs e)
+        {
+            controls.CheckBoxOnlyNumber(e);
+        }
+
+        private void RepeatTimesBox_KeyUp(object sender, KeyRoutedEventArgs e)
+        {
+            controls.BoxFocusOut(e, ControllerPage);
+        }
+
+        private async void RepeatTimesBox_LostFocus(object sender, RoutedEventArgs e)
+        {
+            await controls.BoxesMaxNumberLimiter(3, RepeatTimesBox, FrameSpeedBox, DelayBox);
+            await jsonSaverAndLoader.JsonAutoSaver(framesList, RepeatTimesBox.Text);
+        }
+        #endregion
+
+        #region CONTROLS
+        private async void SendButton_Click(object sender, RoutedEventArgs e)
+        {
+            await httpRequests.GetRequest(Convert.ToString(Eixo1Slider.Value), Convert.ToString(Eixo2Slider.Value), Convert.ToString(Eixo3Slider.Value), Convert.ToString(Eixo4Slider.Value), Convert.ToString(GarraSlider.Value));
```

Comment:
Utilizar a classe Movement proposta?
```suggestion
            await httpRequests.GetRequest(movement);
```